lib/src/identity_codec.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:convert';
+
+class _IdentityConverter<T> extends Converter<T, T> {
+  @override
+  T convert(T input) => input;
+}
+
+/// A [Codec] that converts from T to T doing absolutely nothing.
+///
+/// This is the identity codec, the one that passes input to output when
```

Comment:
/// The identity codec passes input directly to output in both directions.
/// This class can be used as a base when combining multiple codecs,
/// because fusing the identity codec with any other codec gives the other
/// codec back.